pkg/payload/task.go

```diff
@@ -188,6 +188,9 @@ func SummaryForReason(reason, name string) string {
 	case "ImageVerificationFailed":
 		return "the image may not be safe to use"
 
+	case "UpgradePreconditionCheckFailed":
+		return "the update might not be safe to proceed"
```

Comment:
Wording on this is a little weird - maybe `it may not be safe to apply this update`

Reply:
+1 update to match your suggestion.